docs/pages/api-reference/operators.md

```diff
@@ -19,10 +19,12 @@ against another dataset which has keys defined. The join operation is performed
 The `join` operator has the following parameters:
 
 1. `dataset: Dataset` - positional argument, that specifies the RHS Dataset.&#x20;
-2. `on: List[str]` - kwarg that specifies the list of fields to join on.
-3. `left_on: List[str]` - optional kwarg specifying the list of fields to join on for the LHS dataset.&#x20;
-4. `right_on: List[str]` - optional kwarg specifying the list of fields to join on for the RHS dataset.&#x20;
-5. `within: Tuple[Duration, Duration]` - optional kwarg specifying the time window within which the join should be
+2. `how: str` - required kwarg indicating whether the join should be an inner join (`how='inner'`) or a left-outer
+   join (`how='left'`).
+3. `on: List[str]` - optional kwarg that specifies the list of fields to join on.
+4. `left_on: List[str]` - optional kwarg specifying the list of fields to join on for the LHS dataset.&#x20;
+5. `right_on: List[str]` - optional kwarg specifying the list of fields to join on for the RHS dataset.&#x20;
```

Comment:
Should we specify that we either provide on or provide left_on and right_on ?

Reply:
that's called out below:
<img width="728" alt="image" src="https://github.com/fennel-ai/client/assets/877038/b489ee30-127c-4d70-9365-97bed2ee8a0e">